### PR TITLE
index.js default filename

### PR DESCRIPTION
**Pull request by shapkarin**
Changes issue #55

Changes: use "index.js" as default filename

### CODE REVIEW

File: test_folder/components/Test/index.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
```
Comment (rajikaimal) [inline test_folder/components/Test/index.js]: test_folder. I thought we are generating this programmatically only when running unit tests.

Comment (shapkarin) [reply test_folder/components/Test/index.js]: OK, I think the same.

Comment (shapkarin) [reply test_folder/components/Test/index.js]: I create that with cli from terminal. Just to show.. I'll remove that. Not a problem.

Comment (shapkarin) [reply test_folder/components/Test/index.js]: Just saved that accidentally when used cli manually.

Comment (rajikaimal) [reply test_folder/components/Test/index.js]: oh cool

**Pull request merged**